### PR TITLE
Don't create history objects for empty ListField and StringField

### PR DESCRIPTION
**Pull request by ajayaa**

### CODE REVIEW

File: django_model_changes/changes.py [hunk 1]
```diff
@@ -46,15 +48,25 @@ def _calculate_changes(self, created=False, _changed_fields=None, _original_valu
...
             _original_values = getattr(self, '_original_values', {})
+
+        _force_changed_fields = getattr(self, '_force_changed_fields', set())
             
         if created:
             _changed_fields = self._data.keys()
 
         res = {}
-        for field in _changed_fields:
+        for field in set(_changed_fields) | _force_changed_fields:
```
Comment (ajayaa) [inline django_model_changes/changes.py]: Unrelated commit from @sauravshah which was checked out locally in my submodule. Lemme know if it needs to be removed.

Comment (sauravshah) [reply django_model_changes/changes.py]: This shouldn't be removed -- not sure why it's showing up as a diff?

**Comment by addy007-icy** [comment]
https://github.com/Rippling/django-model-changes/pull/14
Incorporated changes here.

**Comment by ajayaa** [comment]
@addy007-icy got these changes in the commit https://github.com/Rippling/django-model-changes/commit/aa9e3cdcee159d4586270ce6356dd263ab77a9d9
